src/core/bittorrent/peerinfo.h

```diff
@@ -89,12 +91,16 @@ namespace BitTorrent
         qlonglong totalDownload() const;
         QBitArray pieces() const;
         QString connectionType() const;
+        qreal relevance() const;
 #ifndef DISABLE_COUNTRIES_RESOLUTION
         QString country() const;
 #endif
 
     private:
+        qreal calcRelevance(const TorrentHandle *torrent) const;
```

Comment:
Doesn't make sense to do so. It should be void non-const method that directly sets the value of ```m_relevance``` variable.